### ISSUE TITLE
Receiving and sending "chat" messages

### ISSUE THREAD

**Issue by Spurlex** [JorgenPhi/php-snapchat#70]
Hi,

I am wondering if anyone has any information on the receiving and sending of chat. 

Would be great if people could post what they have found in regards to this.

**Comment by cuonic** [comment]
Hi,

I'm documenting the new API, I should be adding more of the conversation functions today. Feel free to join in here : https://github.com/JorgenPhi/php-snapchat/wiki/API-v2-Research

**Comment by Spurlex** [comment]
@cuonic Have you worked out how to decrypt + encrypt the chat messages to send them at all?

**Comment by cuonic** [comment]
No, you need to decompile the APK and take some time out browsing the 
source code to work this out, afterwards it will be trivial.

**Comment by Spurlex** [comment]
It would be great if we could have any input from any previous findings that people have found out.

@cuonic @hako

**Comment by cuonic** [comment]
@Spurlex I don't think anyone else is researching this as the chat system is relatively new. I've had a quick look at the encryption method, it uses the "SlightSecurePreferences" class.
